README.md

```diff
@@ -673,6 +687,19 @@ gitMaterial1:
     - src/**/*.*
 ```
 
+<a name="git-material-update"/>
+
+Since GoCD `>= 19.4.0` you can specify username and password as such
+
+```yaml
+gitMaterial1:
+  git: "http://my.git.repository.com"
+  branch: feature12
+  username: my_username
+  password: "some password"
```

Comment:
I think it's probably better that we document `encrypted_password` over `password` :)

Reply:
Yep. Considering the copy-paste rate from these examples.

Reply:
Sure will do that. Also, will update the readme for other materials as well.